pkg/controller/gitopsservice/gitopsservice_controller.go

```diff
@@ -226,6 +227,35 @@ func (r *ReconcileGitopsService) Reconcile(request reconcile.Request) (reconcile
 		}
 	}
 
+	// Ensure compute resource quotas are set for the default ArgoCD namespace
+	resourceQuotaName := argocdNS.Name + "-compute-resources"
+	err = r.client.Get(context.TODO(), types.NamespacedName{Name: resourceQuotaName, Namespace: argocdNS.Name}, &corev1.ResourceQuota{})
+	if err != nil && errors.IsNotFound(err) {
+		reqLogger.Info("Setting compute resource quotas for ArgoCD namespace", "Name", argocdNS.Name)
+		resourceQuota := corev1.ResourceQuota{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "ResourceQuota",
+				APIVersion: "v1",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      resourceQuotaName,
+				Namespace: argocdNS.Name,
+			},
+			Spec: corev1.ResourceQuotaSpec{
+				Hard: corev1.ResourceList{
+					corev1.ResourceMemory:       resourcev1.MustParse("4032Mi"),
+					corev1.ResourceCPU:          resourcev1.MustParse("6188m"),
+					corev1.ResourceLimitsMemory: resourcev1.MustParse("8046Mi"),
+					corev1.ResourceLimitsCPU:    resourcev1.MustParse("12750m"),
```

Comment:
Hi @jopit , Looks like corev1.ResourceLimitsCPU and corev1.ResourceCPU are swapped mistakenly :)

ResourecLimitsCPU should be greater than ResourceCPU.

Reply:
Hi @iam-veeramalla ResourceLimitsCPU is greater than ResourceCPU,  12750m > 6188m

Reply:
Sorry .. my bad @jopit